### DISCUSSION TITLE
Don't display all the localized content items

### DISCUSSION THREAD

**Discussion by hishamco** [OrchardCMS/OrchardCore#13229] [Ideas]
Assume your website support 4 cultures, you have 10 localized articles, so the total will be 40 content items will be displayed in the content items page, which is make it diffcult to manage all the content items from content managers perspective.

I suggest to show only the content item with the default translation, then we can use the `Localizations` button - as we are using today - to manage other cultures for a particular cultures, backing to our example the content items page will show 10 items instead of 40 items which make the management easy IMHO

@Piedone your thoughs while you might have many multilingual websites done by Lombiq

**Comment by Piedone** [comment]
Note that this would require every admin to know the language of the default culture to understand which content item is which. They'd also need to keep the default culture titles of content items in mind (which may be very different from other cultures') when listing them and searching for them on the admin.

I don't think such a change would be beneficial, especially that with the culture filter it's simple to just list items of a single culture.

**Reply by hishamco** [reply]
> Note that this would require every admin to know the language of the default culture to understand which content item is which

IMHO almost the time if not all the default language depends on the owner for the website for instance if the website is Hungary I presume the default is `hu`, if it's Arabic `ar` definitally is `ar`

> I don't think such a change would be beneficial, especially that with the culture filter it's simple to just list items of a single culture.

The filter for sure will reduce the content items, but the issue still exist especailly if you have many localized content items with some supports cultures

**Reply by Piedone** [reply]
Can you elaborate what issue exists, specifically? I thought what you see is having too many content items in the list, but the culture filter solves that.

**Reply by hishamco** [reply]
You 're right Zoltan, I have seen many CMSs are displaying a single content item per localization set with a badges shows which languages are translated into

I'm sure the filter has been added to reduce the amount of displayed content items, while we can reduce the result without the filter
